package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "action-devcard",
-  "version": "2.3.1",
+  "version": "3.0.0",
```

Comment:
Sorry, forgot that I created a workflow for creating a new release, so you can revert this change, and remove the build output as the workflow will automagically create it.

Reply:
Ah, nice!